x/ratelimit/client/cli/query.go

```diff
@@ -22,8 +24,72 @@ func GetQueryCmd() *cobra.Command {
 	}
 
 	cmd.AddCommand(
-		GetCmdQueryQuota(),
-		GetCmdQueryQuotas(),
+		GetCmdQueryRateLimit(),
+		GetCmdQueryRateLimits(),
 	)
 	return cmd
 }
+
+// GetCmdQueryRateLimit implements a command to query a specific rate limit
+func GetCmdQueryRateLimit() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "rate-limit [denom] [channel-id]",
+		Short: "Query a specific rate limit",
+		Args:  cobra.ExactArgs(1),
```

Comment:
I think the number of arguments is set wrongly.
There are same issues for some other query/tx CLIs.
Please check.

Reply:
great catch! fixing now